terraform/groups/provisioning-role/module-iam-policies/terraform-apply.tf

```diff
@@ -0,0 +1,34 @@
+data "aws_iam_policy_document" "apply" {
+  statement {
+    sid       = "ApplyPolicyAllResources"
+    effect    = "Allow"
+    resources = ["*"]
+    actions   = [
+      "ec2:CreateSecurityGroup",
+      "ec2:CreateTags",
+      "lambda:CreateFunction",
+    ]
+  }
+
+  statement {
+    sid       = "ApplyPolicyListedResources"
+    effect    = "Allow"
+    resources = [
+      "arn:aws:iam::${var.aws_account_id}:role/payment-reconciler-execution-*",
+      "arn:aws:events:eu-west-2:${var.aws_account_id}:rule/payment-reconciler-*",
```

Comment:
Region specific, needs to be changed to a variable. This will need to be everywhere, I won't comment on all of them.
Or potentially don't target the ARN specifically and do it via a condition on the name of the resource so it would apply to all regions in one policy.

Reply:
Yeah, I'll change to a variable.
Conditions seem quite hit and miss. Some allow this and sone don't. From what I can see, line 18 here refers to the events resource and therefore the PutRule, EnableRule and PutTargets which do have ec2:ResourceTag condition keys.
It might be simpler to just wildcard the region if we wanted to let it apply to all regions. I was thinking of limiting it to just West 2 though, as that's what we want to do generally.

Reply:
Done for all cases. Replaced hard-coded region with variable set in the profile vars file.